infrastructure/validators/operational-tasks/monitor-node.md

```diff
@@ -0,0 +1,529 @@
+level=info ts=2021-04-16T19:02:20.234Z caller=main.go:975 msg="Completed loading of configuration file" filename=/etc/prometheus/prometheus.yml totalDuration=824.115µs remote_storage=3.131µs web_handler=401ns query_engine=1.056µs scrape=236.454µs scrape_sd=45.432µs notify=723ns notify_sd=2.61µs rules=956ns
+level=info ts=2021-04-16T19:02:20.234Z caller=main.go:767 msg="Server is ready to receive web requests."
+
+```
+
+Go to `http://SERVER_IP_ADDRESS:9090/graph` to check whether you are able to access the Prometheus
+interface or not. If it is working, exit the process by pressing on `CTRL + C`.
+
+To automatically start the server during the boot process, create a new `systemd` configuration file with the following configuration.
+
+```bash
+sudo nano /etc/systemd/system/prometheus.service
+```
+
+```bash
+[Unit]
+  Description=Prometheus Monitoring
+  Wants=network-online.target
+  After=network-online.target
+
+[Service]
+  User=prometheus
+  Group=prometheus
+  Type=simple
+  ExecStart=/usr/local/bin/prometheus \
+  --config.file /etc/prometheus/prometheus.yml \
+  --storage.tsdb.path /var/lib/prometheus/ \
+  --web.console.templates=/etc/prometheus/consoles \
+  --web.console.libraries=/etc/prometheus/console_libraries
+  ExecReload=/bin/kill -HUP $MAINPID
+
+[Install]
+  WantedBy=multi-user.target
+```
+
+Once the file is saved, execute the command below to reload `systemd` and enable the service so that
+it will be loaded automatically during the operating system's startup.
+
+```bash
+sudo systemctl daemon-reload && systemctl enable prometheus && systemctl start prometheus
+```
+
+Prometheus should be running now, and you should be able to access its front again end by
+re-visiting `IP_ADDRESS:9090/`.
+
+## Installing Grafana
+
+In order to visualize your node metrics, you can use Grafana to query the Prometheus server. Run the
+following commands to install it first.
+
+```bash
+sudo apt-get install -y adduser libfontconfig1
+wget https://dl.grafana.com/oss/release/grafana_7.5.4_amd64.deb
+sudo dpkg -i grafana_7.5.4_amd64.deb
+```
+
+If everything is fine, configure Grafana to auto-start on boot and then start the service.
+
+```bash
+sudo systemctl daemon-reload
+sudo systemctl enable grafana-server
+sudo systemctl start grafana-server
+```
+
+You can now access it by going to the `http://SERVER_IP_ADDRESS:3000/login`. The default user and
+password is `admin`.
+
+:::note
+
+If you want to change the port on which Grafana runs (3000 is a popular port), edit the file
+`/usr/share/grafana/conf/defaults.ini` with a command like
+`sudo vim /usr/share/grafana/conf/defaults.ini` and change the `http_port` value to something else.
+Then restart Grafana with `sudo systemctl restart grafana-server`.
+
+:::
+
+![1-grafana-login](/images/infrastructure/validator/operational-tasks/1-grafana-login.webp)
+
+In order to visualize the node metrics, click _settings_ to configure the `Data Sources` first.
```

Comment:
I'd tweak this section a bit to make it look like enumerated steps, I think it would be easier to read